apps/editor/src/js/wwPasteContentHelper.js

```diff
@@ -11,6 +11,8 @@ import matches from 'tui-code-snippet/domUtil/matches';
 import domUtils from './utils/dom';
 import defaultSanitizer from './htmlSanitizer';
 
+const DEFAULT_COLOR = 'rgb(34, 34, 34)';
```

Comment:
Just curious. Why default color is rgb(34,34,34) , not rgb(0,0,0)? and can this be customized?